src/main/java/seedu/address/ui/StatisticsPanel.java

```diff
@@ -146,10 +149,8 @@ private void bindAllValuesToStatistics() {
      * @param newValue the new value of the monthly spent
      */
     private void updateBudgetAdvice(Number newValue) {
-        String adviceText;
-        if (newValue.doubleValue() <= analyticModel.getBudget()) {
-            adviceText = "Great job! You are within your budget!";
-        } else {
+        String adviceText = "Great job! You are within your budget!";
+        if (analyticModel.getBudget() != 0 && newValue.doubleValue() > analyticModel.getBudget()) {
```

Comment:
Will there be further changes to the provided advice in future?

Reply:
Not entirely certain if this will be allowed after the feature freeze, but its stated that as long as there are no UI changes or new features, and this feature is considered part of the Summary Statistics Feature and doesn't involve modifying the UI, might be worth considering updating this in the next milestone.